batch/src/main/scala/uk/gov/ons/addressindex/Main.scala

```diff
@@ -37,9 +38,19 @@ For usage see below:
   val nodes = config.getString("addressindex.elasticsearch.nodes")
   val port = config.getString("addressindex.elasticsearch.port")
 
-  //  each run of this application has a unique index name
-  val indexName = generateIndexName(historical = !opts.hybridNoHist(), skinny = opts.skinny(), nisra = opts.nisra())
-  val url = s"http://$nodes:$port/$indexName"
+ // username and password should be set in the local application.conf when running locally
+ // use the secure Jenkins environment variables for on-prem
+  val username = config.getString("addressindex.elasticsearch.user")
+  val password = config.getString("addressindex.elasticsearch.pass")
+
+  def encodeCredentials(username: String, password: String): String = {
+    new String(Base64.encodeBase64String((username + ":" + password).getBytes))
+  }
+  val authHeader = "Basic " +  encodeCredentials(username, password)
```

Comment:
This is very Jala. Something like this would be nicer!

```
  def encodeCredentials(username: String, password: String): String = {
    Base64.encodeBase64String((s"$username:$password").getBytes)
  }
  val authHeader = s"Basic ${encodeCredentials(username, password)}"
```